ext/opentelemetry-ext-wsgi/src/opentelemetry/ext/wsgi/__init__.py

```diff
@@ -149,12 +149,8 @@ def add_response_attributes(
 
 
 def get_default_span_name(environ):
-    """Calculates a (generic) span name for an incoming HTTP request based on the PEP3333 conforming WSGI environ."""
-
-    # TODO: Update once
-    #  https://github.com/open-telemetry/opentelemetry-specification/issues/270
-    #  is resolved
-    return environ.get("PATH_INFO", "/")
+    """Default implementation for name_callback, returns HTTP {METHOD_NAME}."""
+    return "HTTP " + environ.get("REQUEST_METHOD")
```

Comment:
This could lead to an error if `REQUEST_METHOD` is not defined:

```python
In [2]: "sdf" + environ.get("sdf")
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-2-a24682c43732> in <module>
----> 1 "sdf" + environ.get("sdf")

TypeError: can only concatenate str (not "NoneType") to str
```

Reply:
It could be a good thing to test the default behavior of `get_default_span_name` too.

Reply:
Thanks for the heads up!